documentation/ideas.md

```diff
@@ -127,3 +127,16 @@ Brain dump of all ideas and random thoughts for this app. Many are pipe dreams o
   - Provide the ability for the captain/moderator to leave a custom message
     - back to inprogress example: "Almost finished the dishes but you forgot to start the machine"
     - marked failed example: "Didnt feed the dog and I already did it"
+
+# Crews
+- Since we have the concept of crews now, maybe we want members to be able to belong to other crews
+  - User story: Pippa can belong to the crew at my house, but maybe Jessey's parents want to setup a crew and Pippa can belong there too
```

Comment:
Interesting idea. Will take some thought to figure out how to make that work, but I like it.